public/html/api-spec.html

```diff
@@ -11,7 +11,7 @@
   <body>
 
     <elements-api
-      apiDescriptionUrl="https://gist.githubusercontent.com/dhichii/a660ca04401584cba08ed54e4a563965/raw/1dc3e58909c0f04646db36066152bf9104379a1a/clinic-openapi.yml"
+      apiDescriptionUrl="https://gist.githubusercontent.com/dhichii/a660ca04401584cba08ed54e4a563965/raw/387b4d3d21ec6f97169d841134c49a858459e82d/clinic-openapi.yml"

```

Comment:
Please change this to: <https://raw.githubusercontent.com/ALTERA-CAPSTON-41/backend/master/libs/api-spec/api-spec.yml>

Reply:
don't change, will be resolved at #39.